server/belga/macros/set_default_metadata.py

```diff
@@ -12,7 +12,7 @@
 
 
 logger = logging.getLogger(__name__)
-SUBJECT_SCHEMES = ('news_services', 'news_products', 'distribution')
+SUBJECT_SCHEMES = ('services-products', 'news_services', 'news_products', 'distribution')
```

Comment:
It's fine for now to keep `'news_services', 'news_products'`, but after ingests/output will be migrated to use only `'services-products'` we should delete `'news_services', 'news_products'`.

Reply:
Yes I would have deleted them, but the ticket description asked to keep them.